metadata-models/src/main/pegasus/com/linkedin/structured/StructuredPropertyDefinition.pdl

```diff
@@ -24,6 +24,9 @@ record StructuredPropertyDefinition {
     * The value type of the property. Must be a dataType.
     * e.g. To indicate that the property is of type DATE, use urn:li:dataType:datahub.date
     */
+    @Searchable = {
+      "fieldType": "URN"
+    }
```

Comment:
if we filter here by type, we need to do a backfill job for existing properties.

Reply:
created a ticket to solve for this because this is a bug right now